README.md

```diff
@@ -35,12 +42,22 @@ OR with SSH:
 copier copy git@github.com:cagov/caldata-infrastructure-template.git .
 ```
 
+### Azure DevOps repo
+Install git credential manager (with [Homebrew](https://brew.sh/) if on a mac, if on a windows you should have it by default.) Then run the following three commands:
```

Comment:
I don't think you have git by default on windows, typically windows users need to install git from [here](https://git-scm.com/downloads/win). But when they do install it from there, it comes with the credential manager already

Reply:
Thank you for the correction!